mrb_config.rb

```diff
@@ -113,7 +113,12 @@
     l.library_paths << File.expand_path('soil', bvd)
     l.library_paths.uniq!
 
-    l.libraries << 'glfw'
+    unless Platform.darwin?
+      l.libraries << 'glfw' 
```

Comment:
I think the new header is `'glfw3'` on all platforms now, so just:

```ruby
l.libraries << 'glfw3'
```

That should fix the Travis build